arangod/RestHandler/RestIndexHandler.cpp

```diff
@@ -193,13 +197,89 @@ RestStatus RestIndexHandler::getIndexes() {
       return RestStatus::DONE;
     }
 
+    std::stringstream ap;
+    ap << "Plan/Collections/" << _vocbase.name() << "/" << coll->planId()
+       << "/indexes";
+    auto& ac = _vocbase.server().getFeature<ClusterFeature>().agencyCache();
+    auto [plannedIndexes, idx] = ac.get(ap.str());
```

Comment:
I think we must not execute this code if we are not on a coordinator. At least it should not be executed on a single server, most likely also not on DB servers.